### ISSUE TITLE
[Feature Request] Category Genre Tags for Roms

### ISSUE THREAD

**Issue by DudeShift** [cizia64/CrossMix-OS#60]
Feature: To be able to sort search by category genre type ie puzzle, racing, sports etc. Data can be scraped from the new screenscrapper addition to CrossMix.  

If not possible, the ability to add custom folders in the Best per emulator so that a user could manually add shortcuts to roms that they know/scrape the genre themself.

Reasoning: It seems like the idea behind the Best tab was to make a favorites by based on Emulator to help quickly launch roms without having to go searching in the main list. Having the ability to refined the search down by genre helps those who are in the mood for a certain genre.

Having custom folders per emulator in Best would also allow for this (even though its more work on the users part) but it would help them quickly find what they are looking for. 

Please let me know if I can explain this better.

Tl;Dr: Add genre sort option by getting genre from screenscrapper OR allow the file structure of a "Best" collection to have custom folder names after emulator name (Ie Best->GBA->Puzzle, Racing, Sports, etc folders -> rom names

**Comment by cizia64** [comment]
Hi, I have this idea to use best section as a real "best section" indeed 😄 That's why I have created the "[shortcut example](https://github.com/cizia64/CrossMix-OS/tree/main/Best/Free%20Games%20Collection/Roms/VB)"  in the "Free Games Collection".

So technically it's possible: we could use the rom menu (when you press X on a rom) to add a game to a Best collection. But it would require an UI with a keyboard to enter a collection name or to select an existing one.

So there is some work before making it possible but I keep the idea ;)
